seeders/ex.json

```diff
@@ -0,0 +1,62 @@
+[
```

Comment:
Maybe we should include a title and a topic array?
`"title" : "Compute Factorial",
"topics" : ["factorial", "recursion"...],